python/nano/src/bigdl/nano/utils/tf/backend.py

```diff
@@ -59,8 +59,13 @@ def run_subprocess(self, target, args=..., nprocs=1, envs=None) -> Any:
                 for key, val in os.environ.items():
                     if key not in envs[i]:
                         envs[i][key] = val
+                if 'OMP_NUM_THREADS' in envs[i]:
+                    set_op_thread = envs[i]['OMP_NUM_THREADS']
+                else:
+                    # 0 means the system picks an appropriate number
+                    set_op_thread = '0'
                 ex_list.append(subprocess.Popen([sys.executable, f"{cwd_path}/subprocess_worker.py",
-                                                 temp_dir], env=envs[i]))
+                                                 temp_dir, set_op_thread], env=envs[i]))
```

Comment:
@MeouSker77 please have a look at the changes in `backend.py` and `subprocess_worker.py`. It looks good to me but I may miss something.

Reply:
Note: This change is for stock tensorflow since they are not built with openmp and may have some thread control issue by using OMP_NUM_THREADS